core/bin/dev

```diff
@@ -23,6 +23,6 @@ for p in $GROUPAROO_PIDS; do
 done
 
 "$NODEMON" -e js,jsx,ts,tsx --signal SIGTERM --ignore dist --watch ./src --watch ../plugins/**/*.js \
---exec "$TS_NODE" --transpile-only --log-error src/grouparoo.ts \
+--exec node --inspect -r 'ts-node/register/transpile-only' src/grouparoo.ts \
```

Comment:
@evantahler 
We don't technically need to make this optional since if you don't open the inspector it doesn't do anything new. However I can make it based off a `DEBUG=1` or something if you'd like.